maintenance/2021-02-07-source-fio.md

```diff
@@ -0,0 +1,8 @@
+## source.foundries.io maintenance
+
+On Sun, 2021-02-07 between 13:00 and 15:00 UTC Foundries.io team will be performing scheduled maintenance on the main source-code service [source.foundries.io](https://source.foundries.io).
+
+During the maintenance, service might be unavailable up to 2 hours for any opeations: read and write (pull and push).
+CI system will be affected as well: jobs will time out or be cancelled, and will need to be restarted manually after the maintenance.
```

Comment:
I think you can drop the "CI system ...". If code doesn't come in, CI wouldn't get triggered.

Reply:
But what if somebody commits something in 1-2 minutes before the maintenance?

Reply:
ah. i guess there's a small window between code-pushed, CI tries to pull down that code, that a job could be affected. Maybe just put "might be affected".

Reply:
fixed